ush/config_defaults.yaml

```diff
@@ -2356,92 +2411,27 @@ global:
   #
   #-----------------------------------------------------------------------
   #
-  # Set default ad-hoc stochastic physics options.
-  # For detailed documentation of these parameters, see:
-  # https://stochastic-physics.readthedocs.io/en/ufs_public_release/namelist_options.html
+  # Switches for using ad-hoc stochastic physics options.
+  #
+  # For specific namelist settings associated with these switches, see
+  # stochastic_params.yaml
   #
   #-----------------------------------------------------------------------
   #
   DO_SHUM: false
   DO_SPPT: false
   DO_SKEB: false
-  ISEED_SPPT: 1
```

Comment:
All these stochastic parameters are now mapped directly to their namelist settings in stochastic_params.yaml.